microsoft-edge/webview2/release-notes.md

```diff
@@ -70,18 +70,21 @@ Async methods:
 
 
 <!-- ====================================================================== -->
-## 1.0.1722.32
+## 1.0.1722.45
 
-Release Date: April 10, 2023
+Release Date: April 13, 2023
 
-[NuGet package for WebView2 SDK 1.0.1722.32](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1722.32)
+[NuGet package for WebView2 SDK 1.0.1722.45](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1722.45)
 
-For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 112.0.1722.32 or higher.
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 112.0.1722.45 or higher.
 
 
 <!-- ------------------------------ -->
 #### General
 
+> [!IMPORTANT]

```

Comment:
https://learn.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning - ""

"Important" isn't rendering correctly in GH preview view (here or in 790-prerelease section).  Paradoxically, emphasis-alerts cause info to disappear from awareness as if it jumped out from the page entirely.  I am considering an alternative solution which is already established: a heading with clear wording.